packages/palette/src/elements/Skip/Skip.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import styled from "styled-components"
+import { Clickable, ClickableProps } from "../Clickable"
+import { Text } from "../Text"
+import { visuallyHiddenMixin, visuallyUnhiddenMixin } from "../VisuallyHidden"
+
+const Container = styled(Clickable)`
+  display: block;
+  text-decoration: underline;
+  ${visuallyHiddenMixin}
+
+  &:focus {
+    outline: 0;
+    ${visuallyUnhiddenMixin}
+  }
+`
+
+/** SkipProps */
+export type SkipProps = ClickableProps
```

Comment:
It's unclear what `Skip` pertains to, and seems more specifically to the a subcomponent in the `Carousel` folder.